client/src/dummyData.js

```diff
@@ -0,0 +1,28 @@
+export const dummyEntryData = [
+  {
+    date: 'April 6, 2017',
+    text: 'Test entry'
+  },
+  {
+    date: 'April 3, 2017', 
+    text: 'Test entry'
+  },
+  {
+    date: 'March 9, 2017',
+    text: 'Test entry'
+  },
+  {
+    date: 'March 4, 2017',
+    text: 'Test entry'
+  },
+  {
+    date: 'February 22, 2017',
+    text: 'Test entry'
+  }
+];
+
+export const dummyTimelineData = {
```

Comment:
Will you be writing the function to convert the EntryData into TimelineData later?